eventually/authentication/models.py

```diff
@@ -0,0 +1,145 @@
+"""
+This file contains models for the postgresql database.
+"""
+
+#pylint: disable=W0223
+#The pylint comment above is for ignoring the error below
+#Method 'get_full_name' is abstract in class 'AbstractBaseUser'/'get_short_name'
+#but is not overridden (abstract-method)
+
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.utils.dateformat import format
```

Comment:
Do you need this package?

Reply:
django.utils.dateformat is needed to properly format datetime. Replace it with another solution?

Reply:
int(self.created_at.timestamp()*1000)

Reply:
Fixed. Unnecessary import removed.